src/plugins/home/public/application/components/home_app.js

```diff
@@ -57,9 +57,11 @@ export function HomeApp({ directories, solutions }) {
     addBasePath,
     environmentService,
     telemetry,
+    homeConfig,
   } = getServices();
   const environment = environmentService.getEnvironment();
   const isCloudEnabled = environment.cloud;
+  const isOlly = homeConfig.newHomepage;
```

Comment:
Lets remove Olly specific references in the OSD code and make it more generic since that is creating a psuedo dependency on Olly from OSD core which is bad.

This can be renamed as `isNewHomePageEnabled`.

Reply:
+1. ive started created a backlog of generalizing this component